### PR TITLE
Initial Dashbaord, SC move to Dufour

### PR DESCRIPTION
**Pull request by mjadach-iv**

### CODE REVIEW

**Review by ItsLuized — CHANGES_REQUESTED** [review]

File: config.ts [hunk 1]
```diff
@@ -14,15 +14,16 @@ export const HOPR_TOKEN_USED_CONTRACT_ADDRESS = wxHOPR_TOKEN_SMART_CONTRACT_ADDR
...
+export const HOPR_NODE_STAKE_FACTORY = '0x098B275485c406573D042848D66eb9d63fca311C';
+export const HOPR_NODE_MANAGEMENT_MODULE = '0xB7397C218766eBe6A1A634df523A1a7e412e67eA';
+export const HOPR_NODE_SAFE_REGISTRY = '0xe15C24a0910311c83aC78B5930d771089E93077b';
+export const HOPR_NETWORK_REGISTRY = '0x582b4b586168621dAf83bEb2AeADb5fb20F8d50d';
 
 //Subgraphs
 export const STAKE_SUBGRAPH = 'https://api.studio.thegraph.com/query/40439/hopr-stake-all-seasons/v0.0.10';
-export const STAKING_V2_SUBGRAPH = 'https://api.studio.thegraph.com/query/40438/subgraph-dufour/version/latest';
-//export const STAKING_V2_SUBGRAPH = 'https://api.studio.thegraph.com/proxy/40438/test-dufour/version/latest';
+export const STAKING_V2_SUBGRAPH = 'https://api.studio.thegraph.com/query/40439/hopr-nodes-dufour/version/latest';
```
Comment (diegoalzate) [inline config.ts:24]: I see that this url is for dufour dev environment, but the contracts are production contracts. Does this subgraph index does contracts as well?

Comment (diegoalzate) [reply config.ts:24]: My bad, this subgraph works

Comment (mjadach-iv) [reply config.ts:24]: It should work fine

File: src/router.tsx [hunk 2]
```diff
@@ -283,12 +292,21 @@ const LayoutEnhanced = () => {
...
   const loginData = useAppSelector((store) => store.auth.loginData);
   const [searchParams] = useSearchParams();
   const apiEndpoint = searchParams.get('apiEndpoint');
   const apiToken = searchParams.get('apiToken');
 
+  useEffect(() => {
+    if (environment === 'web3') {
+      document.title = "HOPR | Staking Hub"
+    } else if (environment === 'node') {
+      document.title = "HOPR | Staking Hub"
```
[the remark this answers is not in the document]
Comment (mjadach-iv) [reply src/router.tsx]: Done

**Review by ItsLuized — APPROVED** [review]

**Pull request merged**